### PR TITLE
SAK-40011 - Grades in download Excel File are wrong in Assignment "Group

### PR DESCRIPTION
**Pull request by adrianmticarum**
Submission - One submission per group"

### CODE REVIEW

File: assignment/impl/src/java/org/sakaiproject/assignment/impl/GradeSheetExporter.java [hunk 1]
```diff
@@ -233,7 +233,12 @@ public void getGradesSpreadsheet(final OutputStream out, final String reference,
 
                                                 if (grade == null) {
                                                     // TODO originally called submission.getGradeForUser(userId);
-                                                    grade = submissionSubmitter.getGrade() != null ? submissionSubmitter.getGrade() : submission.getGrade();
+                                                	if(submissionSubmitter.getGrade() != null) {
```
Comment (ern) [inline assignment/impl/src/java/org/sakaiproject/assignment/impl/GradeSheetExporter.java]: There shouldn't be a difference between the assignments grade value and the submitters grade value meaning they both should have the same format.

Comment (adrianmticarum) [reply assignment/impl/src/java/org/sakaiproject/assignment/impl/GradeSheetExporter.java]: Can you please confirm if it is ok now?

**Comment by ern** [comment]
Need to resolve conflicts, also probably would make sense to rearrange that logic a little to be less repetitive!

**Comment by adrianmticarum** [comment]
@ern I've updated the changes you required, could you please take a look? Thanks!

**Comment by adrianmticarum** [comment]
I've just added two more changes to the PR. Please @juanjmerono can you check the new changes?

**Pull request merged**